src/devices/Gpio/Drivers/RockPi4BPlusDriver.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for the Rock Pi 4B Plus.
```

Comment:
edit one of the README files, i.e. https://github.com/dotnet/iot/tree/main/src/devices/Gpio and add mention of this driver in there with a link.

Don't worry about any perf measurements, no need to do that

Reply:
you mean to add them inside `RockchipDriver` group or as new group?

I don't see any OrangePi driver in this readme

Reply:
change it as you feel makes most sense, just make sure it's mentioned somewhere

Reply:
something like that?
https://github.com/dotnet/iot/pull/1918/commits/291c00920dd73916b61f0412c8d027391a19ddef